core/src/test/java/com/alibaba/nacos/core/cluster/MemberUtilTest.java

```diff
@@ -40,7 +40,7 @@
 @RunWith(MockitoJUnitRunner.class)
 public class MemberUtilTest {
     
-    private static final String IP = "1.1.1.1";
+    private static final String IP = "1.1.1.2";
```

Comment:
Why do this change?

Reply:
The unit test in this file cannot pass due, I fix it through changing the IP.

Reply:
But the unit test can pass in CI. Has something in your environment cause cannot pass?

Reply:
My local environment can pass, and I have not change anything about this. It's so confused me,I will check and retry later.